guidelines/releases/release-process.md

```diff
@@ -4,58 +4,78 @@ This document describes how to create a Kyma release using Prow.
+   - `kyma/kyma-docs.yaml`
+   - `kyma/kyma-integration.yaml`
+   - `kyma/kyma-gke-integration.yaml`
+   - `kyma/kyma-gke-upgrade.yaml`
+   - `kyma/kyma-artifacts.yaml`
+   - `kyma/kyma-installer.yaml`
+   - `kyma/kyma-github-release.yaml`
 
-   - for every test
-   - `kyma-docs`
-   - `kyma-integration`
-   - `kyma-gke-integration`
-   - `kyma-gke-upgrade`
-   - `kyma-artifacts`
-   - `kyma-installer`
-
-    >**NOTE:** Read [here](https://github.com/kyma-project/test-infra/blob/master/docs/prow/release-jobs.md) how to define a release job for a component.
+        > **Note:** Remember about modifying `presets` array. For example, for 0.9 release the preset is the following:
 
-    To see all release jobs for the 0.6 release, look for job names with the `pre-rel06` prefix.
-    Since every job name has to be unique, prefix it with `pre-rel{XY}`.
-    Define the release jobs on the `master` branch since Prow reads the job configuration from the `master` branch of the `test-infra` repository.
+        ```
+        - labels:
+            preset-target-commit-0.9: "true"
+            env:
+            - name: RELEASE_TARGET_COMMIT
+                value: release-0.9
+        ```
 
-2. Ensure that tests for release jobs exist. Release tests usually iterate through all release versions and run tests for them.
+1. Ensure that tests for release jobs exist. Release tests usually iterate through all release versions and run tests for them.
 See the `TestBucReleases` test defined in `development/tools/jobs/kyma/binding_usage_controller_test.go` as a reference.
-To add tests for all jobs for the new release, update the `GetAllKymaReleaseBranches()` function
+
+1. Update the `GetAllKymaReleaseBranches()` function
 defined in the `development/tools/jobs/tester/tester.go` file under the `test-infra` repository.
 
-3. Define branch protection rules for the release branch in the `prow/config.yaml` file.
-    For example, see the release-0.6 definition:
+1. Define branch protection rules for the release branch in the `prow/config.yaml` file.
+
+    For example, see the release-0.9 definition:
 
     ```yaml
-    release-0.6:
+    release-0.9:
       protect: true
       required_status_checks:
         contexts:
-          - pre-rel06-kyma-integration
-          - pre-rel06-kyma-gke-integration
-          - pre-rel06-kyma-artifacts
-          - pre-rel06-kyma-installer
+          - pre-rel09-kyma-integration
+          - pre-rel09-kyma-gke-integration
+          - pre-rel09-kyma-artifacts
+          - pre-rel09-kyma-installer
+          - pre-rel09-kyma-gke-upgrade
+          - pre-rel09-kyma-gke-central-connector
     ```
 
-## Release
+### Remove previous release jobs
+
+After adding new release jobs, remove all previous ones. For example, while doing 0.9 release, add `pre-rel09` jobs and remove all `pre-rel06` jobs. Make sure that there are only defined jobs with `pre-rel07`, `pre-rel08` and `pre-rel09` prefixes.
```

Comment:
Hmm, so you don't remove ALL the previous jobs as such but you leave those from three releases back, right?

Reply:
Yes, exactly

Reply:
```suggestion
After adding new release jobs, remove the old ones. Remember to leave jobs for three latest releases. For example, during the preparation for the 0.9 release, add `pre-rel09` jobs and remove all `pre-rel06` jobs. Make sure that the only defined jobs are those with `pre-rel07`, `pre-rel08`, and `pre-rel09` prefixes.
```